compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Popup.skiko.kt

```diff
@@ -512,7 +512,10 @@ private fun rememberPopupMeasurePolicy(
                 -platformInsets.top.roundToPx()
             )
             val positionInWindow = popupPositionProvider.calculatePosition(
-                boundsWithoutInsets, sizeWithoutInsets, layoutDirection, contentSize
```

Comment:
Please revert unrelated change

Reply:
It's not entirely unrelated, and it improves readability, and I don't want to waste time creating a separate PR for such a tiny change.